qemu/tests/zero_copy.py

```diff
@@ -52,6 +52,9 @@ def enable_zerocopytx_in_host(enable=True):
         enable_zerocopytx_in_host(False)
 
     error.context("Boot vm with 'vhost=on'", logging.info)
+    if params.get("nettype") == "user":
+        raise error.TestNAError("Unable start test with user networking,"
```

Comment:
In short, should use `test.cancel()` instead.

API reference:
http://avocado-framework.readthedocs.io/en/53.0/api/test/avocado.html#avocado.Test.cancel